Source/SwiftLintFramework/Rules/FileLengthRule.swift

```diff
@@ -40,7 +40,7 @@ public struct FileLengthRule: ParameterizedRule {
 
     public let example = RuleExample(
         ruleName: "File Line Length Rule",
-        ruleDescription: "Files should be less than 400 lines.",
```

Comment:
In `validateFile(_:)`, I'm using `parameters.first!.value` and I think we should do the same for rule examples.